packages/flutter/lib/src/widgets/preferred_size.dart

```diff
@@ -34,6 +33,134 @@ abstract class PreferredSizeWidget implements Widget {
 /// affect the child's layout in any way. It just advertises a preferred size
 /// which can be used by the parent.
 ///
+/// ### Let's see 2 examples
```

Comment:
I think it's better to stick to a single sample for this widget. You could take the first example, make the Preferred size larger than usual, use a container with the gradient, add a couple of buttons. I think that would be simple enough as an example but also interesting enough to show why you would use this widget.